gene_finder.py

```diff
@@ -1,14 +1,17 @@
 # -*- coding: utf-8 -*-
 """
-YOUR HEADER COMMENT HERE
+Finds amino acid sequences coded by DNA input
 
-@author: YOUR NAME HERE
+@author: Vivien Chen
 
 """
 
 import random
 from amino_acids import aa, codons, aa_table   # you may find these useful
 from load import load_seq
+dna = load_seq("./data/X73525.fa")
```

Comment:
you usually want to separate any function calls from these import statements. It would make more sense to put it under  if __name__ == "__main__":